pyproject.toml

```diff
@@ -40,7 +40,7 @@ boto3 = "~1.19.5"  # (always keep in sync w/ optional dependency)
 coverage = "~6.2"
 django-debug-toolbar = "*"
 flake8 = ">=3.9.2"
-hvac = "~0.11.0"  # (always keep in sync w/ optional dependency)
+hvac = ">=0.11.0, <1.1.0"  # (always keep in sync w/ optional dependency)
```

Comment:
I think this would need a corresponding update to line 31, i.e. `poetry add --optional hvac....`?

Reply:
Good catch, thanks Glenn, Happy Monday!